### PR TITLE
document ``jobs_log_expiration`` setting

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: blackbox/docs/configuration.txt [hunk 1]
```diff
@@ -701,6 +701,17 @@ Collecting Stats
...
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the duration how long jobs kept in the ``sys.jobs_log``
+  table. It periodically clears entries that are finished and older than the
+  specified time period. This setting overrides ``jobs_log_size`` if set and
+  logs a warning message.
```
Comment (mikethebeer) [inline blackbox/docs/configuration.txt]: how to disable the setting?

File: blackbox/docs/configuration.txt [hunk 2]
```diff
@@ -701,6 +701,18 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the duration how long jobs kept in the ``sys.jobs_log``
```
Comment (m-kharbat) [inline blackbox/docs/configuration.txt]: how about this phrasing ? 

This setting specifies the period during which the jobs are kept in the ``sys.jobs_log`` table.

File: blackbox/docs/configuration.txt [hunk 3]
```diff
@@ -701,6 +701,18 @@ Collecting Stats
...
+  table. It periodically clears entries that are finished and older than the
+  specified time interval. This setting overrides ``jobs_log_size`` and logs a
+  warning message if both settings are set explicitly. If the value is set to
+  ``0`` the setting gets disabled.
```
Comment (m-kharbat) [inline blackbox/docs/configuration.txt]: `the setting gets disabled.` does this mean that no job information will be collected ? 
if so maybe it should be said explicitly

File: blackbox/docs/configuration.txt [hunk 4]
```diff
@@ -701,6 +701,23 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the period which the jobs are kept in the ``sys.jobs_log``
```
Comment (m-kharbat) [inline blackbox/docs/configuration.txt]: I think it should be `the period during which`

Comment (m-kharbat) [inline blackbox/docs/configuration.txt]: otherwise LGTM

File: blackbox/docs/configuration.txt [hunk 5]
```diff
@@ -701,6 +701,23 @@ Collecting Stats
...
+  table. It periodically clears entries that are finished and older than the
```
Comment (mikethebeer) [inline blackbox/docs/configuration.txt]: Entries that are finished and older than the specified time interval are cleaned periodically.

Comment (bogensberger) [inline blackbox/docs/configuration.txt]: CrateDB periodically removes expired entries.

File: blackbox/docs/configuration.txt [hunk 6]
```diff
@@ -701,6 +701,23 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
```
Comment (mikethebeer) [inline blackbox/docs/configuration.txt]: remove default

Comment (mikethebeer) [inline blackbox/docs/configuration.txt]: Default: `0s`

**Comment by allisonsparrow** [comment]
This setting specifies the period when jobs are stored in the ``sys.jobs_log`` table. It periodically clears entries that are completed and older than the specified time interval. This setting overrides ``jobs_log_size`` and logs a warning message if both settings are set explicitly. If the value is set to
 ``0`` the setting gets disabled.

**Comment by allisonsparrow** [comment]
If both setttings ``stats.jobs_log_size`` and ``stats.jobs_log_expiration`` are
 disabled, no job information will be collected.

**Pull request merged**